### PR TITLE
COM-139: Add Comet UX Library Icons

### PR DESCRIPTION
**Pull request by jomunker**
This PR adds new icons from the Comet UX library. It also replaces existing icons with new versions and deprecates the icons `Pool`, `Pool2`, `Vignette1`, `Vignette2`, `Betrieb` and `LogischeFilter`.

## Acceptance Criteria
- [x] Add changeset

### CODE REVIEW

File: packages/admin/admin-icons/src/Betrieb.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { SvgIcon, SvgIconProps } from "@mui/material";
+import * as React from "react";
+
+/**
+ * @deprecated The 'Betrieb' icon will be removed in the next release.
```
Comment (thomasdax98) [inline packages/admin/admin-icons/src/Betrieb.tsx:5]:
```suggestion
 * @deprecated The 'Betrieb' icon will be removed in v6.
```

File: packages/admin/admin-icons/src/LogischeFilter.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { SvgIcon, SvgIconProps } from "@mui/material";
+import * as React from "react";
+
+/**
+ * @deprecated The 'LogischeFilter' icon will be removed in the next release.
```
Comment (thomasdax98) [inline packages/admin/admin-icons/src/LogischeFilter.tsx:5]:
```suggestion
 * @deprecated The 'LogischeFilter' icon will be removed in v6.
```

File: packages/admin/admin-icons/src/Pool2.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { SvgIcon, SvgIconProps } from "@mui/material";
+import * as React from "react";
+
+/**
+ * @deprecated The 'Pool2' icon will be removed in the next release.
```
Comment (thomasdax98) [inline packages/admin/admin-icons/src/Pool2.tsx:5]:
```suggestion
 * @deprecated The 'Pool2' icon will be removed in v6.
```

File: packages/admin/admin-icons/src/Vignette2.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { SvgIcon, SvgIconProps } from "@mui/material";
+import * as React from "react";
+
+/**
+ * @deprecated The 'Vignette2' icon will be removed in the next release.
```
Comment (thomasdax98) [inline packages/admin/admin-icons/src/Vignette2.tsx:5]:
```suggestion
 * @deprecated The 'Vignette2' icon will be removed in v6.
```

File: .changeset/real-cars-impress.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin-icons": minor
```
[the remark this answers is not in the document]
Comment (jomunker) [reply .changeset/real-cars-impress.md:2]: Don't know actually 🤔 I'll ask Mitch

**Review by thomasdax98 — DISMISSED** [review]

**Review by johnnyomair — CHANGES_REQUESTED** [review]
Some icons (e.g. Frame80) need to be updated before merging

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**